contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus-thanos.libsonnet

```diff
@@ -57,8 +57,56 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
     thanosPeerService:
       local thanosPeerPort = servicePort.newNamed('cluster', 10900, 'cluster');
       service.new('thanos-peers', { 'thanos-peer': 'true' }, thanosPeerPort) +
+      service.mixin.metadata.withNamespace($._config.namespace) +
       service.mixin.spec.withType('ClusterIP') +
       service.mixin.spec.withClusterIp('None'),
 
+    thanosStoreStatefulSet:
+      local statefulSet = k.apps.v1beta2.statefulSet;
+      local volume = statefulSet.mixin.spec.template.spec.volumesType;
+      local container = statefulSet.mixin.spec.template.spec.containersType;
+      local containerEnv = container.envType;
+      local containerVolumeMount = container.volumeMountsType;
+
+      local labels = { app: 'thanos', 'thanos-peer': 'true' };
+
+      local c =
+        container.new('thanos-store', 'improbable/thanos:v0.1.0') +
```

Comment:
should the container image be `$._config.imageRepos.thanos + ':' + $._config.versions.thanos` instead? (like the `thanosQueryContainer` for example)